labs/src/main/scala/org/scalalabs/basic/lab01/OOExercise.scala

```diff
@@ -40,6 +40,36 @@ import scala.language.implicitConversions
  *   of type [[org.scalalabs.basic.lab01.CurrencyConverter]]
  * - Use the implicit CurrencyConverter to do the conversion. 
  */
-class Euro {
 
-}
\ No newline at end of file
+class Euro(var euro:Int, var cents:Int = 0) extends Currency() {
+  def inCents:Int = euro + cents
+  
+  def +(euroTwo:Euro):Euro = {
+    new Euro(euro + euroTwo.euro, cents + euroTwo.cents)
+  }
+  
```

Comment:
This method body is the kind of thing that many Scala programmers will put one line (and omit the braces and result type), like so:

```def +(euroTwo: Euro) = new Euro(euro + euroTwo.euro, cents + euroTwo.cents)```